pyproject.toml

```diff
@@ -78,6 +78,7 @@ dependencies = [
   "black",
   "isort",
   "ruff",
+  "pydocstringformatter",
```

Comment:
can we get a test executable to locally run the docstringformatter? like test-examples, test-docs etc

Reply:
`pre-commit run --all-files` ?